package.json

```diff
@@ -52,7 +52,27 @@
     "testRegex": "jsfmt\\.spec\\.js$",
     "testPathIgnorePatterns": [
       "tests/new_react",
-      "tests/more_react"
+      "tests/more_react",
+      "tests/typescript_nosemi/jsfmt.spec.js",
+      "tests/typescript/conformance/types/abstractKeyword/jsfmt.spec.js",
+      "tests/typescript/conformance/classes/classDeclarations/classHeritageSpecification/jsfmt.spec.js",
+      "tests/typescript/compiler/jsfmt.spec.js",
+      "tests/typescript/conformance/types/typeReference/jsfmt.spec.js",
+      "tests/typescript/conformance/types/variableDeclarator/jsfmt.spec.js",
+      "tests/typescript/conformance/types/typeParameter/jsfmt.spec.js",
+      "tests/typescript/custom/computedProperties/jsfmt.spec.js",
+      "tests/typescript/conformance/types/functions/jsfmt.spec.js",
+      "tests/typescript/custom/new/jsfmt.spec.js",
+      "tests/typescript/custom/call/jsfmt.spec.js",
+      "tests/typescript/custom/typeParameters/jsfmt.spec.js",
+      "tests/typescript/conformance/internalModules/importDeclarations/jsfmt.spec.js",
+      "tests/typescript/conformance/expressions/functionCalls/jsfmt.spec.js",
+      "tests/typescript/conformance/es6/Symbols/jsfmt.spec.js",
+      "tests/typescript/conformance/classes/jsfmt.spec.js",
+      "tests/typescript/conformance/classes/constructorDeclarations/constructorParameters/jsfmt.spec.js",
+      "tests/typescript/custom/module/jsfmt.spec.js",
+      "tests/function/jsfmt.spec.js",
```

Comment:
I think we should just remove `"typescript"` from the list of parsers in that file, rather than disabling the function tests for the flow parser as well.